### DISCUSSION TITLE
What is server-control used for?

### DISCUSSION THREAD

**Discussion by boroczkigabor** [gekkekoe/esphome-ecodan-hp#66] [General]
Hi,

First of, thanks for your work! I just switched from another implementation to yours and it provides much more features. Awesome!

This server-control thing is not clear to me though, what are these supposed to be doing exactly? When to use them? Would you mind explaining it a bit?
Thank you!

**Comment by gekkekoe** [comment]
Server control is a mode where you do things manually. The schedules are ignored for example.
When enabled, you can set the prohibits for heating/cooling/dhw. It gives you full control by enabling what/when you want.

It can be used to enable dhw for example when energy is low.

**Reply by boroczkigabor** [reply]
Ah okay, thanks - so keeping it simple I disabled all of those 🙂

**Comment by jamessimpson007** [comment]
I've read elsewhere that as there is no way to exit Server Control Mode from the FTC6, it could be difficult to get out of in the event that communication via the CN105 is lost. If this were the case, would power-cycling my unit at the fuse get back into default mode? Am I worrying about nothing?

**Reply by F1p** [reply]
The FTC will kick you out of Server Control Mode when CN105 communications timeout/stop

**Comment by jamessimpson007** [comment]
OK thanks!



From: Phil Thomson ***@***.***>
Date: Thursday, 3 April 2025 at 13:46
To: gekkekoe/esphome-ecodan-hp ***@***.***>
Cc: jamessimpson007 ***@***.***>, Comment ***@***.***>
Subject: Re: [gekkekoe/esphome-ecodan-hp] What is server-control used for? (Discussion #66)

The FTC will kick you out of Server Control Mode when CN105 communications timeout/stop
